webpack/assets/javascripts/react_app/components/Editor/__tests__/Editor.test.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { act } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
```

Comment:
Is it needed to rewrite the test to another library for this bugfix?

Reply:
It was added by mistake. Removed from the PR.